openapi_tester/type_declarations.py

```diff
@@ -39,14 +37,10 @@ def export_anything(cls, module_name: str) -> None:
     We don't have to worry about circular imports here, since
     the type checker does not actually run the code.
     """
-    # noinspection PyUnresolvedReferences
-    from rest_framework.response import Response
-
-    # noinspection PyUnresolvedReferences
-    from rest_framework.test import APITestCase
+    from rest_framework.response import Response  # noqa: F401
```

Comment:
this would be needed basically for every line. Its not good. You can disable the rule for the entire file, which is better.

I need to noinspection flags for intellij idea because it complains about unused imports. If you can turn this rule for the entire file, it would be good.

Reply:
So the noinspection flags aren't necessary for me when there's a `noqa` flag, so I thought it would be better to have `n` noqa comments than `2n + 1`  lines of code to manage the noqa file comment + all the noinspect lines.

If the inline `noqa` doesn't work for you, lets revert :)

Reply:
I see the same as you on my home comp, so reverting 👍

Reply:
so this is up to intellij inspection settings. Some of the insepections are global, and i dont want to turn these off. The imports one seems to be like this (tell me if you find a better way.) --> p.s. we can also have a .idea folder to share settings, if its helpful.